tests/e2e/specs/accessView.js

```diff
@@ -1,6 +1,6 @@
 import { createSelection } from '../utils'
 
-describe('Land correctly in every view accesible from the home page', () => {
+describe.skip('Land correctly in every view accesible from the home page', () => {
```

Comment:
Are these skipped tests intentional?

Reply:
We are working on that. There is an open issue to solve this